scripts/west_commands/runners/nrfutil.py

```diff
@@ -97,7 +111,7 @@ def _exec_batch(self):
         self._ops = []
         self._op_id = 1
         self.logger.debug(f'Executing batch in: {json_file}')
-        self._exec(['execute-batch', '--batch-path', f'{json_file}',
+        self._exec(['x-execute-batch', '--batch-path', f'{json_file}',
```

Comment:
please rebase, this has been fixed in another PR

Reply:
done